tests/test_autodoc.py

```diff
@@ -510,14 +545,14 @@ def test_autodoc_members(app):
     # default (no-members)
     actual = do_autodoc(app, 'class', 'target.inheritance.Base')
     assert list(filter(lambda l: '::' in l, actual)) == [
-        '.. py:class:: Base',
+        '.. py:class:: Base()',
```

Comment:
Note: `py:class` directive skips empty parenthesis. So this change harms nobody.

Reply:
Excellent news.